testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/organization/broker/AbstractBrokerSelfRegistrationTest.java

```diff
@@ -785,9 +785,9 @@ public void testRememberOrganizationWhenReloadingLoginPage() {
         assertFalse(loginPage.isSocialButtonPresent(realmIdp.getAlias()));
 
         driver.navigate().back();
-        // only org idp, back button won't reset the flow and the organization is the same
```

Comment:
@pedroigor why clicking back should not return us back and restart the flow? It is the case in firefox and chrome web drivers, only htmlUnit doing weird things here.